contracts/L1GlobalForkRequester.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+/* 
+   Contract to proxy a fork request from the bridge to its ForkingManager
+   Any L2 contract can call us over the bridge to get us to request a fork.
+   We record the dispute they were forking over
+*/
+
+// TODO: An alternative would be to look up the ForkingManager on L2ChainInfo and call it directly.
```

Comment:
Is this viable, if we have to bundle two different transactions: claimAsset and claimMessage? I don't think so, we need this middleware, right?